src/__tests__/fixtures/firestore.rules

```diff
@@ -1,12 +1,30 @@
 service cloud.firestore {
   match /databases/{database}/documents {
     match /users/{userID} {
-      allow read: if true;
-      allow update: if isCurrentUser(userID);
+      allow read: if resource.data != null && resource.data.public == true;
+      allow write: if isCurrentUser(userID);
+
+      match /companies/{companyID} {
+          allow read: if isPublicUser(userID);
+          allow update: if isCurrentUser(userID);
+      }
+    }
+
+    match /settings/{userID} {
+        allow read: if isCurrentUser(userID);
+        allow write: if (
+            isCurrentUser(userID) &&
+            getAfter(/databases/$(database)/documents/users/$(userID)) != null
```

Comment:
Related to my question about result of `get` and `getAfter`, is that the right way that the other document exist ?

Is there a `existAfter` function ?

Reply:
No but `existAfter` would be nice!  I'll ask about that,